src/textual/drivers/headless_driver.py

```diff
@@ -48,7 +59,8 @@ def send_size_event():
         send_size_event()
 
     def disable_input(self) -> None:
-        pass
+        """Start application mode."""
```

Comment:
I don't get it. The methdo `disable_input` also starts application mode..?

Reply:
Cut n paste error